js/pictures-preview.js

```diff
@@ -10,12 +10,14 @@ function createPictureElement (elementData) {
   pictureElement.querySelector('.picture__img').src = elementData.url;
   pictureElement.querySelector('.picture__comments').textContent = elementData.comments.length;
   pictureElement.querySelector('.picture__likes').textContent = elementData.likes;
-  pictureElement.addEventListener('click', function () {
-    showBigPicture(elementData);
-  });
+  pictureElement.addEventListener('click', onPictureClick);
   return pictureElement;
 }
 
+function onPictureClick () {
+  showBigPicture(this.elementData);
+}
+
 function createPictures (picturesData) {
```

Comment:
```suggestion
function createPictures (pictures) {
```

дата лишнее, итак понятно, что передаём данные